### DISCUSSION TITLE
problem in installing the foundry

### DISCUSSION THREAD

**Discussion by Basha-dude** [Cyfrin/foundry-full-course-cu#406] [Q&A] [answered]
giving this error : curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 131066 ms: Connection timed out
when written this  command  curl -L https://foundry.paradigm.xyz | bash

**Comment by ChefAharoni** [comment]
Try the suggestions stated in [this discussion](https://www.debugpoint.com/failed-connect-raw-githubusercontent-com-port-443/).

**Comment by Equious** [comment, marked as answer]
You can check out this discussion as well:
https://github.com/Cyfrin/foundry-full-course-f23/discussions/233
